examples/helm-charts/cubestore/templates/workers/serviceaccount.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.workers.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "cubestore.workers.serviceAccountName" . }}
```

Comment:
I might be missing something, and this is actually the way it's done. Genuinely asking about it! 😄